### PR TITLE
Add --ca-cert arg to encrypt-ami

### PR DESCRIPTION
**Pull request by bonzotronix**
* New suppressed arg for --ca-cert specifies a file to use for the Yeti CA cert. That file contents gets added to the brkt-files part.
* Changed 'description' arg to 'help' arg for one of the add_argument() calls, to prevent an error.
* Addressed feedback from PR #308.

Testing:
* Ran 'encrypt-ami' with and without new arg.
* Ran unit tests.
* Ran 'encrypt-ami' with bogus CA cert.

### CODE REVIEW

File: brkt_cli/user_data.py [hunk 1]
```diff
@@ -155,7 +155,29 @@ def to_mime_text(self):
         return str(container)
 
 
-def combine_user_data(brkt_config=None, proxy_config=None, jwt=None):
```
Comment (bburtin) [inline brkt_cli/user_data.py]: I believe that we've been putting brkt_config stuff in util.py.

Comment (bburtin) [inline brkt_cli/user_data.py]: At this rate, I'll be adding `brkt_config.py` before long...

File: brkt_cli/user_data.py [hunk 2]
```diff
@@ -185,6 +207,15 @@ def combine_user_data(brkt_config=None, proxy_config=None, jwt=None):
             BRKT_FILES_CONTENT_TYPE
         )
 
+    if ca_cert:
+        domain = get_domain_from_brkt_config(brkt_config)
+        ca_cert_filename = "/var/brkt/ami_config/ca_cert.pem.%s" % (domain)
```
Comment (bburtin) [inline brkt_cli/user_data.py]: Not sure what the metavisor code is doing with the filename.  Is it potentially brittle to assume that the domain is everything past the first dot?

Comment (bonzotronix) [inline brkt_cli/user_data.py]: This follows the same logic as Metavisor (specifically the pinYetiCertAuthority function in Wildling's identity server).

File: brkt_cli/user_data.py (hunk 1 above)
Comment (bonzotronix) [inline brkt_cli/user_data.py]: Hee.  That's where it was in the last PR (#308) and you asked me to put it in user_data.  But I forgot to take it out of util.py, so I'll just leave it there and remove the duplicate from user_data.py.

**Comment by bonzotronix** [comment]
Superseded by #326

**Pull request closed without merging**